scripts/tm_vec_run.py

```diff
@@ -168,12 +197,15 @@
 
 #Alignment section
 #If we are also aligning proteins, load tm_vec align and align nearest neighbors if true
-if args.align == True:
-    align_model = LightningAligner.load_from_checkpoint(args.tm_vec_align_path)
+if args.deepblast_model is not None:
+    align_model = DeepBLAST.load_from_checkpoint(
+        args.deepblast_model, lm=model, tokenizer=tokenizer,
+        alignment_mode=args.alignment_mode,  # TODO
+        device=args.device)
     align_model = align_model.to(device)
 
     #Read in database sequences
-    with open(args.database_sequences) as handle_db:
+    with open(args.database) as handle_db:
```

Comment:
@tymor22 I'm getting errors here.  I'm a little confused here. Do we really need to loop through all of the sequences in the database?

Reply:
If we are really passing in the database sequences separately, we also need to make all of those fasta files available as well.

Reply:
This has been resolved